Assets/Scripts/KeyController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
try to give namespaces before the class it can differentiate the functionality related classes